server.js

```diff
@@ -39,14 +43,34 @@ app.get("/menu_items", function (req, res) {
   ]);
 });
 
-//Post Method
-app.post("/menu_items", (req, res, next) => {
-  res.send(JSON.stringify(req.body));
+app.post("/food", async (req, res) => {
+  const food = await prisma.food.create({ data: req.body });
+  res.json(food);
```

Comment:
The `PUT` request looks so nice now.  You should consider applying the same approach here.  More specifically, cherry-picking the `title`, `img`, and `author` fields from the body instead of trusting that the body will always be correct.

Reply:
Updating and will be part of the next PR